products/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Sure Shot Products</title>
+    <link rel="stylesheet" href="../common/main.css">
+    <link rel="stylesheet" href="../product/index.html">
+</head>
+<body>
+    <header>
+        <nav>
+            <a href="../">Home</a>
+        </nav>
+    </header>
+    <main>
+        <h2>Cameras</h2>
+        <ul id="cameras">       
+        </ul>
+    </main>
+    <script src="./app.js" type="module"></script>
```

Comment:
The file you actually want to import here is products.js. That's where you're looping through the data and rendering each camera.